tests/queries/0_stateless/02884_create_view_with_sql_security_option.sh

```diff
@@ -0,0 +1,226 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+
+user1="user02884_1_$RANDOM$RANDOM"
+user2="user02884_2_$RANDOM$RANDOM"
+user3="user02884_3_$RANDOM$RANDOM"
+db="db02884_$RANDOM$RANDOM"
```

Comment:
Why isn't this using the database that's created for each test? `CLICKHOUSE_DATABASE`.

Having it's custom database makes things harder to track and clean up (specially since the test is slow and might be killed in local envs).

Reply:
Also we could use this idea even for objects which are not databases:

```
user2="user2_${CLICKHOUSE_DATABASE}_$RANDOM"
```